app/i18n/en-US/settings.json

```diff
@@ -22,11 +22,11 @@
   "Your cache directory has been successfully uploaded.  The file name %{file} has been copied to your clipboard.": "Your cache directory has been successfully uploaded.  The file name %{file} has been copied to your clipboard.",
   "Streamlabels session has been succesfully restarted!": "Streamlabels session has been succesfully restarted!",
   "This action will restart the application. Continue?": "This action will restart the application. Continue?",
-  "Show confirmation dialog when starting streams": "Show confirmation dialog when starting streams",
```

Comment:
I think this is intentional.  When referring to popup windows in computer programs, `dialog` is actually the most common spelling.

Reply:
See: https://www.merriam-webster.com/dictionary/dialog%20box